ee/tabby-db/migrations/0025_github-provided-repos-vendor-id-index.up.sql

```diff
@@ -0,0 +1 @@
+CREATE INDEX `idx_github_provided_repositories_vendor_id` ON github_provided_repositories(vendor_id);
```

Comment:
add unique constraint as well?

I also suggest merge this with migration 0024 for simplification (it breaks some compatibility, but should be tolerable for developers)